Application/utils.py

```diff
@@ -83,7 +83,11 @@ def setupLxmertTransformer():
 
     # Define the model
     lxmert_tokenizer = LxmertTokenizer.from_pretrained("unc-nlp/lxmert-base-uncased")
-    lxmert_vqa = LxmertForQuestionAnswering.from_pretrained("unc-nlp/lxmert-vqa-uncased")
+
+    # *** OLD MODEL. Load this if no local model is present.
+    # lxmert_vqa = LxmertForQuestionAnswering.from_pretrained("unc-nlp/lxmert-vqa-uncased") 
```

Comment:
We want to have the base and the fine tuned model loaded into the GUI. It looks this line is removing the base model. 
You might have to look at adding the finetuned model around here in the code: https://github.com/jenniferolenchak/DroneVQA/blob/main/Application/application.py#L73